k8s-charm-sidecar/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Penelope Valentine Gale
```

Comment:
You have this copyright notice in multiple files under your name but on k8s-charm-sidecar/charmcraft.yaml for example is `# Copyright 2022 Canonical Ltd.`. Shouldn't be Canonical Ltd everywhere?